tests/docker/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+docker-compose up --build -d
+docker-compose exec web bash -c "composer install"
+docker-compose exec web bash -c 'echo "<?php \$config[\"databases\"][\"mysql\"][\"dsn\"] = \"mysql:host=db;dbname=yiitest;\"; \$config[\"databases\"][\"mysql\"][\"password\"] = \"travis\" ?>" > vendor/yiisoft/yii-core/config/tests/config.local.php'
```

Comment:
This should be done with the config plugin

Reply:
I thought it, but it would have overwritten default db config (in yii-core) everytime

Reply:
I think that what `start.sh` does should be done in a `Dockerfile` instead.

```dockerfile
FROM yiisoftware/yii-php:7.1-fpm

WORKDIR /var/www/html/
RUN composer install
...
```

Reply:
I didn't want to add Dockerfile but using only docker-compose.yml, to keep config as simple as possible.

Reply:
It's possible to do this in a `Dockerfile` but then you don't really have the files in your working directory and code-completion is usually missing. If it is just for testing it might be OK.